source/blood/src/view.cpp

```diff
@@ -3233,7 +3233,7 @@ void viewUpdateDelirium(void)
     if ((powerCount = powerupCheck(gView, kPwUpDeliriumShroom)) != 0)
     {
         int tilt1 = 170, tilt2 = 170, pitch = 20;
-        int timer = (int)gFrameClock*4;
+        int timer = (int)gFrameClock << 1;
         if (powerCount < 512)
```

Comment:
Just multiplying by 2 instead of 4 but by bit shifting one position, even if this is unnecessary optimization hehe.

Reply:
I believe there is also a possibility that something else is modifying the clock upstream before it gets to this code, though I didn't see anything that looked wrong (I might be wrong). Also a possibility the clock class (`ClockTicks`) is newer so maybe it doesn't behave exactly like the original game's clock did? Nonetheless I don't believe anybody has reported any other issues with things that the clock is used for, also the code for `ClockTicks` used in EDuke is identical (hasn't had anything tweaked or fixed), so it seems like it's more prudent to make the change locally here (at where the clock value ends up being used), rather than somewhere upstream or in the clock code itself...